source/qml/Library/RendererQml/NumberInputRender.cpp

```diff
@@ -0,0 +1,429 @@
+#include "NumberInputRender.h"
+#include "Formatter.h"
+#include "utils.h"
+#include "ImageDataURI.h"
+
+NumberinputElement::NumberinputElement(std::shared_ptr<AdaptiveCards::NumberInput> input, std::shared_ptr<RendererQml::AdaptiveRenderContext> context)
+    :mInput(input),
+    mContext(context),
+    numberConfig(mContext->GetRenderConfig()->getInputNumberConfig())
+{
+}
+
+std::shared_ptr<RendererQml::QmlTag> NumberinputElement::getDummyElementforNumberInput(bool isTop)
+{
+    auto DummyTag = std::make_shared<RendererQml::QmlTag>("Rectangle");
+    DummyTag->Property("width", "2");
+    DummyTag->Property("height", "2");
+    DummyTag->Property("anchors.right", "parent.right");
+
+    if (isTop)
+    {
+        DummyTag->Property("anchors.top", "parent.top");
+    }
+    else
+    {
+        DummyTag->Property("anchors.bottom", "parent.bottom");
+    }
+    DummyTag->Property("anchors.margins", "5");
+    DummyTag->Property("color", "'transparent'");
+    DummyTag->Property("z", "-1");
+
+    return DummyTag;
+}
+
+std::shared_ptr<RendererQml::QmlTag> NumberinputElement::getQmlTag()
+{
+    if (mInput->GetHeight() == AdaptiveCards::HeightType::Stretch)
+    {
+        return RendererQml::AdaptiveCardQmlRenderer::GetStretchRectangle(numberInputColElement);
```

Comment:
Do we need GetStretchRectangle() function?

Since we are having Column as the outer element i think we can neglect this function since column will take care of this function's purpose